### ISSUE TITLE
The accuracy doesn't change 

### ISSUE THREAD

**Issue by morco6** [bearpelican/musicautobot#22]
Why while training the accuracy doesn't change? 
Even in the example train notebook it stays on 0.397959..

**Comment by asigalov61** [comment]
I would check batch size, model config/code and also training data...

Check out my fork of Andrew's notebooks...it should be working fine...

**Comment by morco6** [comment]
I didnt touch nothing.. maybe my data.
tnx for the first response, cool project!
